code/game/objects/items/melee/misc.dm

```diff
@@ -326,6 +334,197 @@
 	cooldown = 10
 	stamina_damage = 20
 
+//Wooden Baton
+/obj/item/melee/classic_baton/police/beater
+	name = "wooden baton"
+	desc = "Favored by hot headed Security Officers who don't want to get in trouble with CentCom but still want to get that nostalgic feeling of beating some criminal scum upside the head with a chunk of wood."
+	force = 8
+	throwforce = 7
+	cooldown = 0
+	stamina_damage = 15
+	stun_animation = TRUE
+	/// Per-mob sleep cooldowns.
+	/// [mob] = [world.time where the cooldown ends]
+	var/list/sleep_cooldowns = list()
+	/// Per-mob trip cooldowns.
+	/// [mob] = [world.time where the cooldown ends]
+	var/list/trip_cooldowns = list()
+
+/obj/item/melee/classic_baton/police/beater/attack(mob/living/target, mob/living/user)
+	if(!on)
+		return ..()
+	var/def_check = target.getarmor(type = MELEE, penetration = armour_penetration)
+
+	add_fingerprint(user)
+	if((HAS_TRAIT(user, TRAIT_CLUMSY)) && prob(50))
+		to_chat(user, "<span class ='danger'>You hit yourself over the head.</span>")
+		user.adjustStaminaLoss(stamina_damage)
+
+		additional_effects_carbon(user) // user is the target here
+		if(ishuman(user))
+			var/mob/living/carbon/human/H = user
+			H.apply_damage(2*force, BRUTE, BODY_ZONE_HEAD)
+		else
+			user.take_bodypart_damage(2*force)
+		return
+	if(iscyborg(target))
+		// We don't stun if we're on harm.
+		if (user.a_intent != INTENT_HARM)
+			if (affect_silicon)
```

Comment:
Not sure if you copypasted this, but this affecting silicons is kind of insane since seccies already have a flash for this.

Reply:
Coppied this straight from the normal police baton, so whatever this does so does Deputy and Detective baton do!

Reply:
![image](https://github.com/BeeStation/BeeStation-Hornet/assets/94647521/6e6c73d9-09ae-4b5d-a5cf-384ad021373d)
Also, this.

So no, it DOESN'T affect silicons.

Reply:
Might wanna just take the code out, it's additional code for no reason

Reply:
It was there originally for some reason. I'm going to keep it for now until someone else comments over this, since you were wrong at first and I don't have the knowledge necessary to evaluate this well enough.

Reply:
If this baton is not intended to stun silicons, then you shouldn't include code to stun silicons, it's just extra code for no reason.